packages/zoltan/src/include/zoltan_partition_tree.h

```diff
@@ -0,0 +1,112 @@
+ * the U.S. Government retains certain rights in this software.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are
+ * met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ * notice, this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright
+ * notice, this list of conditions and the following disclaimer in the
+ * documentation and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the Corporation nor the names of the
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY SANDIA CORPORATION "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL SANDIA CORPORATION OR THE
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * Questions? Contact Karen Devine	kddevin@sandia.gov
+ *                    Erik Boman	egboman@sandia.gov
+ *
+ * ***********************************************************************
+ *
+ * @HEADER
+ */
+
+#ifndef ZOLTAN_PARTITION_TREE_H
+#define ZOLTAN_PARTITION_TREE_H
+
+#ifdef __cplusplus
+/* if C++, define the rest of this header file as extern C */
+extern "C" {
+#endif
+
+/*****************************************************************************/
+/*
+ *  Function to return the partition tree info generated by RCB.
+ *  Input:
+ *    zz                  --  The Zoltan structure returned by Zoltan_Create.
+ *    treeNodeIndex       --  The array index into the tree
+ *  Output:
+ *    parent              --  parent partition number
+ *    left_leaf           --  left leaf partition number
+ *    right_leaf          --  right leaf partition number
+ *  Returned value:       --  Error code
+ */
+int Zoltan_RCB_Partition_Tree(
+  struct Zoltan_Struct     *zz,
+  int    treeNodeIndex,
+  int    *parent,
+  int    *left_leaf,
+  int    *right_leaf
+);
+
+/*****************************************************************************/
+/*
+ *  Function to get the PHG tree size.
+ *  Input:
+ *    zz                  --  The Zoltan structure returned by Zoltan_Create.
+ *  Output:
+ *    tree_size           --  will be set to the size of the tree
```

Comment:
I've added PR #1165 which explains the parameters and conventions more clearly. This PR is just comments (no code changes).